### DISCUSSION TITLE
Welcome to mojodojo.dev Discussions!

### DISCUSSION THREAD

**Discussion by mojodojodev** [mojodojodev/mojodojo.dev#21] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a way to collect feedback from the community:
  * Share ideas for what you want to see next
  * Give feedback on how and what you want to see improve

Questions about the Mojo language can be asked on [the offical Mojo repo](https://github.com/modularml/mojo/discussions/categories/q-a)

**Comment by gautam-e** [comment]
I really like the recent "intro to Mojo" guide that @mojodojodev made. Would be great to progressively build up on this. 

E.g I'm working on making an MNIST classifier (taking my time with that, became a father for the second time, a couple of weeks ago 😅). 

Is that something we'd want to see here? Or is the intention to stick purely to the Mojo language rather than covering machine learning stuff.

**Reply by mojodojodev** [reply]
@gautam-e  congrats on the new child! 

Definitely would love that type of thing, I do have another side thing I'm working on for another folder in `guides` called `examples`, I'm trying to use a Buffer to directly edit numpy image arrays generated from opencv. But would also like to have real world examples in `Intro to Mojo` as the chapters progress, so a MNIST classifier might work in perfectly, but if it's more specific without introducing new concepts it would definitely have a prominent place in another folder if you like.

**Reply by gautam-e** [reply]
Thanks @mojodojodev! 

Ok sounds great. I look forward to the numpy and opencv example!  
It's a good idea to have these "bite-sized" guides indeed. 
I guess I'll share the example I'm working on when it's somewhat ready for you to have a look.
